game/civ5map.ksy

```diff
@@ -0,0 +1,157 @@
+meta:
+  id: civ5map
+  application: Sid Meier's Civilization V
+  file-extension: civ5map
+  encoding: UTF-8
+  license: CC0-1.0
+  endian: le
+doc: |
+  Many of Civilization 5's maps are in civ5map format, and encodes terrain and
+  sometimes resources on the map plots. Scenario data is also encoded in some
+  civ5map files, although this does not currently parse such data.
+
+  Civ5maps can be accompanied by lua files which defines advanced behavior for
+  world gen on top of what's already in the civ5map.
+
+  There are three versions of the format identified in this file- A, B, and
+  C. The latest version of Civ can read all of them. Version A is the base
+  version. Version B has some additional information about the map in the
+  header (string3). Version C has an additional part in the header that
+  defines which lua files Civ should use when starting a new game with the map.
+doc-ref: https://forums.civfanatics.com/threads/civ5map-file-format.418566/
+seq:
+  - id: header
+    type: header
+  - id: mapdata
+    type: mapdata
+types:
+  header:
+    seq:
+      - id: has_scenario_data
+        type: b4
+      - id: version
+        type: b4
+        
+      - id: width
+        type: u4
+      - id: height
+        type: u4
+      - id: players
+        size: 1
+        
+      - id: misc_settings_head
+        type: b5
```

Comment:
Does it have internal structure?

Reply:
It's always 0s there, since it precedes a three-bit bitwise settings thing

Reply:
Then it probably should documented. Any observations, guesses, speculations and ideas about the content and purpose of other fields which purpose is jot clearly known are also welcome in `doc`.

Reply:
Section is now documented. https://github.com/kaitai-io/kaitai_struct_formats/pull/243/commits/d7bf117bc3ac20f37159109ec667eb3591c6f2ff